ccd-definition/CaseTypeTab.json

```diff
@@ -93,6 +93,17 @@
     "CaseFieldID": "c21Orders",
     "TabFieldDisplayOrder": 3
   },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "CARE_SUPERVISION_EPO",
+    "Channel": "CaseWorker",
+    "TabID": "OrdersTab",
+    "TabLabel": "Orders",
+    "TabDisplayOrder": 4,
+    "CaseFieldID": "caseManagementOrderAction",
+    "TabFieldDisplayOrder": 4,
+    "FieldShowCondition": "caseManagementOrderAction.cmoActionType!=\"SEND_TO_ALL_PARTIES\""
+  },
```

Comment:
just explain this to me? It will display the action if the type isn't send to all parties?

Reply:
how will I see it's been given the ok?